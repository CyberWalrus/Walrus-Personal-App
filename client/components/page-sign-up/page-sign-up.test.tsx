import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import PageSignUp from "./page-sign-up";

Enzyme.configure({ adapter: new Adapter() });
it(`PageSignUp correctly render`, (): void => {
  const tree: Json = toJson(shallow(<PageSignUp />));
  expect(tree).toMatchSnapshot();
});
