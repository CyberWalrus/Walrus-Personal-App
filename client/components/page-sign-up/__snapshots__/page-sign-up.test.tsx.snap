// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSignUp correctly render 1`] = `
<div
  className="page"
>
  <Header />
  <main
    className="page-content"
  >
    <section
      className="form-custom-box"
    >
      <Connect(WithFormState)
        formType="SIGN_UP"
        titel="Sign Up"
      />
    </section>
  </main>
  <Footer />
</div>
`;
